### PR TITLE
[SPARK-23197][DStreams] Increased timeouts to resolve flakiness

### PR DESCRIPTION
**Pull request by tdas**
## What changes were proposed in this pull request?

Increased timeout from 50 ms to 300 ms (50 ms was really too low). 

## How was this patch tested?
Multiple rounds of tests.

### CODE REVIEW

**Comment by tdas** [comment]
@sameeragarwal this PR should fix this flakiness.

File: streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala [hunk 1]
```diff
@@ -105,13 +105,13 @@ class ReceiverSuite extends TestSuiteBase with TimeLimits with Serializable {
     assert(executor.errors.head.eq(exception))
 
     // Verify restarting actually stops and starts the receiver
-    receiver.restart("restarting", null, 100)
-    eventually(timeout(50 millis), interval(10 millis)) {
+    receiver.restart("restarting", null, 600)
```
Comment (sameeragarwal) [inline streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala]: If these timeout bumps fix the flakiness, we should also consider enabling the "block generator throttling" test below (it was disabled via https://github.com/apache/spark/commit/b69c4f9b2e8544f1b178db2aefbcaa166f76cb7a)

**Comment by sameeragarwal** [comment]
LGTM, pending jenkins. Thanks!

File: streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala (hunk 1 above)
Comment (tdas) [reply streaming/src/test/scala/org/apache/spark/streaming/ReceiverSuite.scala]: yeah. lets fix one at a time.

**Pull request closed without merging**